mev_inspect/classifiers/helpers.py

```diff
@@ -55,6 +55,43 @@ def create_swap_from_transfers(
     )
 
 
+def create_swap_from_transfers_not_including_pool(
```

Comment:
Instead of `_not_including_pool` it'd be good to specify the difference

Maybe something like:
`create_swap_from_transfers` => `create_swap_from_pool_transfers`
`create_swap_from_transfers_not_including_pool` => `create_swap_from_recipient_transfers`